### DISCUSSION TITLE
[REGRESSION] ERROR: No files to upload, WARNING: renovate-log.ndjson: no matching files

### DISCUSSION THREAD

**Discussion by mjnowen** [renovatebot/renovate#30592] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab.com/renovate-bot/renovate-runner

### Please tell us more about your question or problem

Hello, I am getting this error since ~v38 MAJOR release:

```
WARNING: renovate-log.ndjson: no matching files. Ensure that the artifact path is relative to the working directory (/builds/<redacted>) 
ERROR: No files to upload
```

I've not changed anything in my Renovate config, but something in the latest (possibly since ~v38 MAJOR release) has broken the ability to upload the artifact at end of the Renovate CI run. See below for comparison between 5 days ago and 4 days ago (I run Renovate once a day, so capture whatever the latest version available is on each daily run).

**"renovateVersion": "37.440.7"**

```
Creating cache main-renovate-26-protected...
renovate/cache/renovate/repository/: found 4 matching artifact files and directories 
Archive is up to date!                             
Created cache
Uploading artifacts for successful job
Uploading artifacts...
renovate-log.ndjson: found 1 matching artifact files and directories 
Uploading artifacts as "archive" to coordinator... 201 Created  id=9007448 responseStatus=201 Created token=glcbt-64
Cleaning up project directory and file based variables
Job succeeded
```

**"renovateVersion": "38.17.1"**

```
Creating cache main-renovate-26-protected...
renovate/cache/renovate/repository/: found 4 matching artifact files and directories 
Archive is up to date!                             
Created cache
Uploading artifacts for successful job
Uploading artifacts...
WARNING: renovate-log.ndjson: no matching files. Ensure that the artifact path is relative to the working directory (/builds/<redacted>) 
ERROR: No files to upload                          
Cleaning up project directory and file based variables
Job succeeded
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
Duplicate of https://gitlab.com/renovate-bot/renovate-runner/-/issues/105

**Reply by viceice** [reply, marked as answer]
fixed in https://gitlab.com/renovate-bot/renovate-runner/-/releases/v18.1.10
